installinstallmacos.py

```diff
@@ -370,17 +379,31 @@ def main():
                                          product_info[product_id]['BUILD'],
                                          product_info[product_id]['title'])
 
-    answer = raw_input(
-        '\nChoose a product to download (1-%s): ' % len(product_info))
-    try:
-        index = int(answer) - 1
-        if index < 0:
-            raise ValueError
-        product_id = product_info.keys()[index]
-    except (ValueError, IndexError):
-        print 'Exiting.'
+    # Quit now if displaynonly argument supplied
+    if args.displayonly:
```

Comment:
Assuming a goal of automated, non-interactive use of the script, I'd think you'd want more machine-parsable output. Perhaps change `--displayonly` to `--list` and then print simpler output that is easier to parse. (My bias towards Python might even lead to a --plist output...)

Reply:
It already works in my use case. But yes, more use cases could be served with a more generic list-based output.